packages/gafl-webapp-service/src/pages/multibuy/remove-licence/__tests__/update-transaction.spec.js

```diff
@@ -53,24 +112,243 @@ describe('remove-licence > update transaction', () => {
     jest.clearAllMocks()
   })
 
-  it('set is being called with a permission removed', async () => {
+  const transaction = {
+    permissions: [getTransactionPermissionOne(), getTransactionPermissionTwo(), getTransactionPermissionThree()]
+  }
+
+  const page = {
+    permissions: [getPagePermissionOne(), getPagePermissionTwo(), getPagePermissionThree()]
+  }
+
+  const status = {
+    permissions: [getStatusPermissionOne(), getStatusPermissionTwo(), getStatusPermissionThree()]
+  }
+
+  const addressLookup = {
+    permissions: [getAddressLookupPermissionOne(), getAddressLookupPermissionTwo(), getAddressLookupPermissionThree()]
+  }
```

Comment:
I've mentioned I don't know how many times not to do this...

"Fresh test, fresh mocks"